posts/2021-11-18-til-cleaner-queries-with-ecto-map.md

```diff
@@ -0,0 +1,75 @@
+%{
+author: "Yuri Oliveira",
+author_link: "https://github.com/yuriploc",
+tags: ["ecto"],
+date: ~D[2021-11-18],
+title: "TIL: Cleaner queries with Ecto `map`",
+excerpt: """
+How learning more about the Ecto.Query module can help you write cleaner select queries.
```

Comment:
I might sum this up in a way that is a bit more specific to this post's content. Maybe something like: "Learn how you can write cleaner Ecto select queries with the help `Ecto.Query.map`"